apps/site/src/app/(main)/apply/Form/ProfileInformation.tsx

```diff
@@ -1,5 +1,8 @@
+import Textfield from "@/lib/components/forms/Textfield";
 import TextInput from "@/lib/components/forms/TextInput";
 
+const FRQ_MAX_LENGTH = 2000;
```

Comment:
Just realized that max length is technically 2048 in the api, but 2000 here

Reply:
Different browsers count newline characters slightly differently, so the limit in the frontend was originally made to be slightly lower than in the API (see description in #152). Not sure if this is still the case, but might be good to explain this as a comment in the code.